modules/abstract-utxo/src/recovery/backupKeyRecovery.ts

```diff
@@ -16,6 +16,9 @@ const {
 type ChainCode = utxolib.bitgo.ChainCode;
 type RootWalletKeys = utxolib.bitgo.RootWalletKeys;
 type WalletUnspent<TNumber extends number | bigint = number> = utxolib.bitgo.WalletUnspent<TNumber>;
+type WalletUnspentJSON<TNumber extends number | bigint = number> = utxolib.bitgo.WalletUnspent<TNumber> & {
```

Comment:
It should be `type WalletUnspentJSON = utxolib.bitgo.WalletUnspent & { valueString: string; };`